client/state/selectors/get-selected-or-all-sites-jetpack-can-manage.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedOrAllSites } from 'state/selectors';
+
+/**
+ * Return an array with the selected site or all sites
+ * filtering the ones Jetpack can manage (equivalent to sitesList.getSelectedOrAllJetpackCanManage)
+ *
+ * @param {Object} state  Global state tree
+ * @return {Array}        Array of Sites objects with the result
+ */
+export default ( state ) => {
+	return getSelectedOrAllSites( state ).filter( ( site ) =>
+			site.jetpack &&
+			site.canManage &&
+			get( site, 'capabilities.manage_options', false )
```

Comment:
I've noticed you're using `get( site, 'capabilities.manage_options', false )`, but for checking capabilities we should be using the `canCurrentUser` selector. That applies for the rest of the capabilities usages within this PR.